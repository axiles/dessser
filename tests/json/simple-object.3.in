{ "foo": 3e1, "bar": { "baz": [1,2,3], "glop": true } }
