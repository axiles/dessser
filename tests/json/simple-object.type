{
  foo: float;
  bar: {
    baz: i32?[]?;
    glop: bool;
  }?;
  zap: i8[2][[]]?;  // list of vectors
}?
