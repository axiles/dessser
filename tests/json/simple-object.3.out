(0x1.ep+4 ((1 2 3) T) null)
